writeengine/wrapper/writeengine.cpp

```diff
@@ -4348,7 +4686,39 @@ int WriteEngineWrapper::updateColumnRecs(const TxnID& txnid,
     if (m_opType != DELETE)
         m_opType = UPDATE;
 
-    rc = writeColumnRecords(txnid, cscColTypeList, colExtentsStruct, colValueList, ridLists, tableOid);
+    for (auto cpInfoP : pointersToInfos)
```

Comment:
It looks like this double setExtentsMaxMin() call is a pattern that I don't get.
If it is really necessary then plz make it a function to reduce boilerplate.